app/lib/steps/income_section.rb

```diff
@@ -35,6 +40,12 @@ def employment_steps(session_data)
         Steps::Group.new(:employment) if Steps::Logic.employed?(session_data)
       end
 
+      def employment_status_step
```

Comment:
I think we want `employment_status` in the same `Steps::Group` as `employment` (the Group system determines whether you go on a multi-screen loop when you click a Change link on the check answers page. But equally, that might be something to sort out in EL-943 when we wire up the new employment screen, so maybe we can ignore that for now.

Reply:
Turns out I'm wrong about this anyway - Kate has confirmed that she doesn't want the two screens to be grouped in this way (and we may eventually remove this whole grouping system on all sections)